### DISCUSSION TITLE
Weekly report 02.11. - 09.11.

### DISCUSSION THREAD

**Discussion by TadaXD20** [SE-TINF22B6/Plapy#80] [Weekly reports]
Hello everyone,
this week was about the implementation of the CI workflow. Because Discord does not allow automated testing we are not able to test the functions yet. But we check automatically if the build is stable:
![Build_test](https://github.com/SE-TINF22B6/Plapy/assets/75337582/6c994a29-1f43-42d8-b686-4a6600dc8450)

Our configuration can be found here: [https://github.com/SE-TINF22B6/Plapy/blob/65f6d5f9bd4b54bdba2e7d3da512299a826c3f48/.github/workflows/node.yml]
# Overview
1. Before merging a team member has to review the pullrequest
2. After approval the testing script will be started and tests if the current state can be build
3. After passing the tests merging will be executed

# Detailed explanation
![CI_config](https://github.com/SE-TINF22B6/Plapy/assets/75337582/14805c43-66ad-41b3-a72f-0e55eedb5582)

# Other achieved goals
- corrected our diagrams from last week

**Comment by LeonmAndreesen** [comment]
Hi, 
i like your detailed week post. The use of pictures helped a lot to understand what is going on. I like how you dealt with the fact that your programm doesn't allow you proper testing. Furthermore i see a clear structure in your project.
